soccer/gameplay/skills/capture.py

```diff
@@ -99,7 +99,8 @@ def on_enter_course_approach(self):
 
     def execute_course_approach(self):
         # don't hit the ball on accident
-        self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        #self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        self.robot.disable_avoid_ball()
```

Comment:
Um, the code says if the target point is not the ball it self go to that point with the ball as an obstacle. Otherwise it says go straight to the ball. 
I can't really do anything when we keep telling the robot to go straight into an obstacle....
I mean I can add some heuristics to keep down replanning, but why do we keep telling our robots to plan paths running straight into obstacles?